tests/v3_api/test_multi_cluster_app.py

```diff
@@ -0,0 +1,194 @@
+        "persistence.existingClaim": "",
+        "persistence.size": "10Gi",
+        "persistence.storageClass": "",
+        "serviceType": "NodePort",
+        "wordpressEmail": "user@example.com",
+        "wordpressPassword": "",
+        "wordpressUsername": "user"
+    }
+}
+
+answer = {
+    "values": {
+        "defaultImage": "true",
+        "externalDatabase.database": "",
+        "externalDatabase.host": "",
+        "externalDatabase.password": "",
+        "externalDatabase.port": "3306",
+        "externalDatabase.user": "",
+        "image.repository": "bitnami/wordpress",
+        "image.tag": "4.9.8-debian-9",
+        "ingress.enabled": "true",
+        "ingress.hosts[0].name": "xip.io",
+        "mariadb.db.name": "wordpress",
+        "mariadb.db.user": "wordpress",
+        "mariadb.enabled": "true",
+        "mariadb.image.repository": "bitnami/mariadb",
+        "mariadb.image.tag": "10.1.35-debian-9",
+        "mariadb.mariadbPassword": "",
+        "mariadb.master.persistence.enabled": "false",
+        "mariadb.master.persistence.existingClaim": "",
+        "mariadb.master.persistence.size": "8Gi",
+        "mariadb.master.persistence.storageClass": "",
+        "nodePorts.http": "",
+        "nodePorts.https": "",
+        "persistence.enabled": "false",
+        "persistence.size": "10Gi",
+        "persistence.storageClass": "",
+        "serviceType": "NodePort",
+        "wordpressEmail": "user@example.com",
+        "wordpressPassword": "",
+        "wordpressUsername": "user"
+    }
+}
+ROLES = ["project-member"]
+# TEMP_VER = "cattle-global-data:library-wordpress-1.0.5"
+TEMP_VER = "cattle-global-data:library-wordpress-2.1.10"
+
+
+def test_multi_cluster_app_create():
+    targets = [{"projectId": namespace["namespace1"]["project_id"],
+                "type": "target"},
+               {"projectId": namespace["namespace2"]["project_id"],
+                "type": "target"}]
+    client = global_client["client"]
+    p_client1 = namespace["namespace1"]["p_client"]
+    p_client2 = namespace["namespace2"]["p_client"]
+    multiClusterApp = client.create_multiClusterApp(templateVersionId=TEMP_VER,
+                                                    targets=targets,
+                                                    roles=ROLES,
+                                                    name=random_name(),
+                                                    answers=[answer])
+    multiClusterApp = wait_for_mcapp_to_active(client, multiClusterApp)
+    app_id1 = multiClusterApp.targets[0].appId
+    app_id2 = multiClusterApp.targets[1].appId
+    id1 = 0
+    id2 = 0
+    if app_id1 is None:
+        id1 = 1
+        print("app_id1 is None")
+    assert id1 == 0
+    if app_id2 is None:
+        id2 = 1
+        print("app_id2 is None")
+    assert id2 == 0
+    # verify if this app is available in the cluster/project
+    multiClusterApp = validate_multi_cluster_app_cluster(app_id1, app_id2,
```

Comment:
Include checks to make sure you are able to launch the wordress app as part of validation.